VRDR.Messaging/Parameters.cs

```diff
@@ -0,0 +1,1277 @@
+
```

Comment:
Is this the file of generated code? If it is, it should have a comment saying it's generated and not to edit because changes will be overwritten

Reply:
To add to this, it may be useful as part of the CI to automate the generation of this file in order to provide something to check against in CI to make sure nobody has manually modified the content. The advantage being that would restrict anyone from accidentally modifying the content of these files and then having their changes lost down the road.